src/Models/EventModel.php

```
@@ -1,292 +1,303 @@
+            error_log("Event insertion successful: " . json_encode($event));
+            // Retrieve the last inserted ID for the event
+            $event->eventID = $db->lastInsertId();
+
+            // Insert each time slot into the `event_tijd` table
+            $sqlEventTime = "INSERT INTO `event_tijd` 
+                             (event_id, land, plaatsnaam, straatnaam, huisnummer, postcode, datum, begin_tijd, eind_tijd) 
+                             VALUES (:eventID, :Land, :Plaats, :Straatnaam, :Huisnummer, :Postcode, :date, :BeginTijd, :EindTijd)";
+
+            $stmtEventTime = $db->prepare($sqlEventTime);
+
+            foreach ($event->eventTime as $i => $slot) {
```

PHPMD (2.15.0)

Avoid unused local variables such as '$i'. (undefined)

(UnusedLocalVariable)